modules/ROOT/pages/push/proc_sending-a-push-notification.adoc

```diff
@@ -0,0 +1,40 @@
+include::{partialsdir}/attributes.adoc[]
+
+//':context:' is a vital parameter. See: http://asciidoctor.org/docs/user-manual/#include-multiple
+:context: proc_sending-a-push-notification
+
+[id='{context}_sending-a-push-notification']
+
+= Sending a Push Notification
+
+The {push-ui} allows you to send {push-notification}s to devices.
+
+[discrete]
+== Prerequisites
+
+* Access to the {push-ui}.
+* An application with at least one binding or variant.
+
+[discrete]
+== Procedure
+
+. Open the {push-ui} in a browser.
+
+. Select the target application from the home page and click *Send Notification To This App*.
+
+. Enter text in the *Message* form.
++
+NOTE: The default priority of a Push Notification is `normal`. To increase the priority, change *Priority* to `high`.
+
+. Click *Send Push Notification* to send the message to the target application.
+
+NOTE: The 3rd party Push Network is responsible for delivering the Push Notification to the target application.
+
+[discrete]
+== Additional Resources
+
+=== Filtering recipients
+
+It is possible to select a more specific set of recipients by adding additional values to *Variants*, *Alias*, *Device Types* and *Category*.
+A {mobile-client} must be associated with one or more of these parameters in order for the filtering to work.
+Filtering is done when you xref:ROOT:push/coding.adoc[Register a Device].
```

Comment:
@jstaffor What is done during the registration is that those parameters (alias, category, etc) are set.

Reply:
Ok, I'll make the update. Re the heading, "=== Filtering recipients" - can you think of a more accurate/descriptive heading?